MetaMorpheus/EngineLayer/SpectralMatch.cs

```diff
@@ -85,7 +85,7 @@ public FdrInfo FdrInfo
         public FdrInfo PeptideFdrInfo { get;  set; }
         public FdrInfo GetFdrInfo(bool peptideLevel)
         {
-            return peptideLevel ? PeptideFdrInfo : PsmFdrInfo;
+            return peptideLevel & PeptideFdrInfo != null ? PeptideFdrInfo : PsmFdrInfo;
```

Comment:
If I call for the peptide level info, and it is null, this will return the psm level info. This feels dangerous